app/holds/remove-hold.js

```diff
@@ -4,7 +4,7 @@ const { sendHoldEvent } = require('../event')
 
 const removeHold = async (holdId) => {
   await db.hold.update({ closed: Date.now() }, { where: { holdId } })
-  const hold = await db.hold.findByPk(holdId)
+  const hold = await db.hold.findOne({ where: holdId, raw: true })
```

Comment:
This should be `raw` so it's a plain JS object, rather than a full sequelize model.  This causes issue downstream in event persistence.